src/TokenRegistrationPage.tsx

```diff
@@ -19,17 +20,18 @@ const TokenRegistrationPage: React.FC = () => {
   const [payeeId, setPayeeId] = useState('');
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState<string | null>(null);
-  const [tokenData, setTokenData] = useState<any | null>(null);
+  const [tokenData, setTokenData] = useState<TtokenData | null>(null);
 
-  const postDataAndReturnToken = async (requestData: any) => {
+  const postDataAndReturnToken = async (requestData: unknown) => {
     try {
-      const apiUrl = process.env.API_URL
-
-      if (!apiUrl) {
-        console.error('One or more required environment variables are not defined');        
-      }
+      console.log('postDataAndReturnToken requestData:', requestData);
+      // const apiUrl = process.env.API_URL
```

Comment:
I think, we can remove this commented code, right?

Reply:
Yes let me also address this.